tgui/packages/tgui/interfaces/Uplink.js

```diff
@@ -0,0 +1,187 @@
+            onClick={() => act('buyRandom')}
+          />
+          <Button
+            content="Refund Currently Held Item"
+            icon="undo"
+            onClick={() => act('refund')}
+          />
+        </Fragment>
+      }>
+      <Flex>
+        <FlexItem>
+          <Tabs vertical>
+            {cats.map(c => (
+              <Tabs.Tab
+                key={c}
+                selected={c === uplinkCat}
+                onClick={() => setUplinkCat(c)}>
+                {c.cat}
+              </Tabs.Tab>
+            ))}
+          </Tabs>
+        </FlexItem>
+        <Flex.Item grow={1} basis={0}>
+          {uplinkCat.items.map(i => (
+            <Section
+              key={i.name}
+              title={i.name}
+              buttons={
+                <Button
+                  content={"Buy (" + i.cost + "TC)"}
+                  color={i.hijack_only === 1 && "red"}
+                  tooltip={i.hijack_only === 1 && "Hijack Agents Only!"}
+                  tooltipPosition="left"
+                  onClick={() => act("buyItem", {
+                    item: i.obj_path,
+                  })}
+                  disabled={i.cost > crystals}
+                />
+              }>
+              <Box italic>
+                {i.desc}
+              </Box>
+            </Section>
+          ))}
+        </Flex.Item>
+      </Flex>
+    </Section>
+  );
+};
+
+
+const ExploitableInfoPage = (_properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    exploitable,
+  } = data;
+  // Default to first
+  const [
+    selectedRecord,
+    setSelectedRecord,
+  ] = useLocalState(context, 'selectedRecord', exploitable[0]);
+
+  const [
+    searchText,
+    setSearchText,
+  ] = useLocalState(context, 'searchText', '');
+
+  // Search for peeps
+  const SelectMembers = (people, searchText = '') => {
+    const MemberSearch = createSearch(searchText, member => member.name);
+    return flow([
+      // Null camera filter
+      filter(member => member?.name),
+      // Optional search term
+      searchText && filter(MemberSearch),
+      // Slightly expensive, but way better than sorting in BYOND
+      sortBy(member => member.name),
+    ])(people);
+  };
+
+  const crew = SelectMembers(exploitable, searchText);
+
+  return (
+    <Section title="Exploitable Records">
+      <Flex>
+        <FlexItem basis={20}>
+          <Input
+            fluid
+            mb={1}
+            placeholder="Search Crew"
+            onInput={(e, value) => setSearchText(value)} />
+          <Tabs vertical>
+            {crew.map(r => (
+              <Tabs.Tab
+                key={r}
+                selected={r === selectedRecord}
+                onClick={() => setSelectedRecord(r)}>
+                {r.name}
+              </Tabs.Tab>
+            ))}
+          </Tabs>
+        </FlexItem>
+        <Flex.Item grow={1} basis={0}>
+          <Section title={"Name: " + selectedRecord.name}>
```

Comment:
Isn't this meant to contain the exploitable info from their char preferences? I don't see age/rank/species/etc as exploitable.

Reply:
It contains the exact same as the old records did, and adding or removing stuff makes this fall outside of refactor territory